src/llnode.cc

```diff
@@ -282,6 +283,23 @@ bool PluginInitialize(SBDebugger d) {
   interpreter.AddCommand("jssource", new llnode::ListCmd(),
                          "Alias for `v8 source list`");
 
+  v8.AddCommand("listobjects", new llnode::ListObjectsCmd(),
+                "List all object types and instance counts grouped by map and "
+                "sorted by instance count.\n"
+                "Requires RANGESFILE to be set to a file containing memory "
+                "ranges for the core file being debugged.");
+
+  interpreter.AddCommand("listobjects", new llnode::ListObjectsCmd(),
```

Comment:
We probably want to call it `findjsobjects` or something like this. These aliases are for people who are used to `mdb_v8`.